openshift/jenkins-persistent-template.json

```diff
@@ -28,7 +28,7 @@
         "tls": {
```

Comment:
@wtrocki - This caused problems in Chrome - for dev purposes can we jus t leave it out for now until we can agree on certificates for aerogear